bots/tests-scan

```diff
@@ -306,9 +345,6 @@ def cockpit_tasks(api, update, contexts, repo):
         have = len(statuses.keys()) > 0
 
         baseline = BASELINE_PRIORITY
-        # modify the baseline slightly to favor older pull requests, so that we don't
-        # end up with a bunch of half tested pull requests
-        baseline += 1.0 - (min(100000, float(number)) / 100000)
```

Comment:
This will remove the aging. If we switch to AMQP soon, this is fine, and we can live without this for a few days. The commit message could point this out, though. Or maybe conditionalize this if `--amqp` is not given?

Reply:
ah yes of course, i'll add that back in